tests/rptest/tests/cluster_bootstrap_test.py

```diff
@@ -110,6 +106,8 @@ def __init__(self, test_context):
     def setUp(self):
         # NOTE: `rpk redpanda admin brokers list` requires versions v22.1.x and
         # above.
+        self.OLDVER = self.installer.latest_for_line((22, 2))
+        self.OLDVER_STR = f"v22.2.{self.OLDVER[2]}"
```

Comment:
I think you need to use something like this for the "after" version as well as the "before" version -- in this patch, the test is still jumping straight from 22.2.x to `dev` (which is an unsupported upgrade path, because it skips `22.3.x`)

Reply:
yes, forgot to commit the lines.
Question: should the upgrade version be the latest_for_line((22, 3)) or should it be v22.3.1? what makes more sense?

Reply:
The latest version from each feature branch makes sense.